### DISCUSSION TITLE
Unique ID Extension: Enable on all types?

### DISCUSSION THREAD

**Discussion by craigrileyuk** [ueberdosis/tiptap#5833] [General Questions] [answered]
So the documentation has 

```js
UniqueID.configure({
  types: ['heading', 'paragraph'],
}),
```

and the default for `type` is an empty array, which adds IDs to nothing.

Is there a way to generate IDs for every type without having to write them out? I need the ID for some functionality regardless of its type.

**Comment by nperez0111** [comment, marked as answer]
It's better to be explicit, adding attributes to everything can be excessive
